cmd/gaiad/genaccounts.go

```diff
@@ -51,7 +52,7 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 			if err != nil {
 				// attempt to lookup address from Keybase if no address was provided
 				kb, err := keys.NewKeyring(
-					sdk.GetConfig().GetKeyringServiceName(),
+					version.Name,
```

Comment:
Why do this when the default is already this (per your PR).

Reply:
Fixed